trulens_eval/trulens_eval/Example_TruBot.py

```diff
@@ -16,16 +16,14 @@
 from trulens_eval.utils.imports import OptionalImports
 from trulens_eval.utils.imports import REQUIREMENT_PINECONE
 
-with OptionalImports(messages=REQUIREMENT_PINECONE):
+with OptionalImports(messages=REQUIREMENT_PINECONE) as opt:
```

Comment:
what's up with aliasing as opt? why do this?

Reply:
to not have to recreate an identical `OptionalImports` instance again when doing `assert_installed` later